packages/metadata/src/decorator-factory.ts

```diff
@@ -328,7 +328,8 @@ export class DecoratorFactory<
       if (v == null) return v;
       if (
         v.constructor != null &&
-        !DecoratorFactory._cloneableTypes.includes(v.constructor)
+        DecoratorFactory._cloneableTypes.indexOf(v.constructor) > -1
```

Comment:
Seems `sdocs` doesn't pick up the `tsconfig`, so `indexOf` is a function of `string` instead of `array`

Reply:
You can use `sdocs --tsconfig ...`

Reply:
@raymondfeng thanks! I will try. I thought it will automatically detect the `tsconfig` from the project. Maybe because we keep those configuration files in the `build` package, it doesn't find it.